redash/monitor.py

```diff
@@ -155,7 +155,7 @@ def rq_queues():
         q.name: {
             'name': q.name,
             'started': fetch_jobs(q, StartedJobRegistry(queue=q).get_job_ids()),
-            'queued': fetch_jobs(q, q.job_ids)
+            'queued': len(q.job_ids)
```

Comment:
Does `q.job_ids` include `queued` and `started`?

Reply:
RQ does an `LPOP` when it starts executing a job so `q.job_ids` won't include started ones (instead, they are moved to `rq:wip:...`, which is observed by the `StartedJobRegistry`).